src/clj_kondo/impl/analyzer.clj

```diff
@@ -2178,8 +2178,10 @@
           ;; analyze-expressions should go first in order to process ignores
           (when (identical? :edn lang)
             (let [fn (.getName (io/file filename))]
-              (when (= fn "deps.edn")
-                (deps-edn/lint-deps-edn ctx (first (:children parsed)))))))))
+              (condp = fn
```

Comment:
why not just use case here?

Reply:
Ya, I dunno, I started with a case and somehow ended up with a condp.  
Will change to case.